javascript/javascript2/week3/js2week3.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Java Script 2 week 3</title>
+    <style>
+        body {
+  background: black;}
+  button {
+  background-color: burlywood;
+  border-radius: 1em;
+  display: inline-block;
+  padding: 2em;
+  transform: scale(.9);
+  transform-origin: 0 0;
+  transition: transform .6s;
+  user-select: none;
+  font-size: 30px;
+}
+
+    </style>
+</head>
+<body id = "dclk">
+<button id="click">click Here</button>
```

Comment:
Looks good. 
1. One thing you could maybe make better is a better naming convention. The id="click" would be confusing in a bigger project. I'd suggest and id="delay5Sec"
2. The id before "dclick" is a better naming convention... though.. the body should not have an id .. but yeah. good work around